pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -2041,6 +2050,16 @@ type Bond struct {
 	BondPeerNotifyDelay uint32 `yaml:"peerNotifyDelay,omitempty"`
 }
 
+// Bridge contains the various options for configuring a bridge interface.
+type Bridge struct {
+	//   description: The interfaces that make up the bond.
+	BridgedInterfaces []string `yaml:"interfaces"`
+	//   description: |
+	//     A bridge option.
+	//     Please see the official kernel documentation.
+	BridgeSTPEnabled bool `yaml:"stpEnabled,omitempty"`
```

Comment:
it looks like `STP` might have many more options so, it might make sense to structure the config accordingly:

```yaml
bridge:
  stp:
     enabled: true
```

so we could add more settings in the future

also it feels like STP should be enabled by default, so I would rather do:

```yaml
bridge:
  stp:
     disabled: false
```

Reply:
(we can still keep more natural logic in `provider.go` interfaces, this is just for YAML zero value being sane default)

Reply:
or another, more natural option be `Enabled *bool` with `nil` being `true`

Reply:
I changed it to be the `Enabled *bool` as you suggested.